src/navsections/K8sResourceSectionBlueprint/K8sResourceSectionEmptyDisplay.tsx

```diff
@@ -46,7 +48,8 @@ function K8sResourceSectionEmptyDisplay() {
 
     setTimeout(() => {
       if (itemToHighlight === HighlightItems.BROWSE_TEMPLATES) {
-        dispatch(setLeftMenuSelection('templates-pane'));
+        dispatch(setActiveSettingsPanel(SettingsPanel.Plugins));
```

Comment:
I think that templates will have a different pane, not the plugins from settings